util/TLSProxy/Proxy.pm

```diff
@@ -535,4 +537,13 @@ sub reneg
     return $self->{reneg};
 }
 
+sub ciphersuite
+{
+    my $class = shift;
```

Comment:
class.  `$ciphersuite` is a class variable (i.e. a module global variable), not an instance variable.  It's the same as `is_tls13` about 10 lines up.